OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -282,6 +282,12 @@ public interface IMove
 		bool CanEnterTargetNow(Actor self, Target target);
 	}
 
+	public interface ITemporaryBlocker
+	{
+		bool CanRemoveBlockage(Actor self, Actor blocking);
```

Comment:
I guess you are referencing the `Actor self` argument? This was done to conform with the ongoing convention to have `self` as first argument. Not really any other reason.